app/code/Magento/Inventory/Test/Acceptance/ActionGroup/FillMainProductFormInMultiSourceModeActionGroup.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="../../../../../../vendor/magento/magento2-functional-testing-framework/src/Magento/FunctionalTestingFramework/Test/etc/actionGroupSchema.xsd">
+    <!--Fill main fields in create product form-->
```

Comment:
Such comment is redundant because we can understand what is happening here from action group name.
I suggest to add action before fillFields:
`<comment userInput="Fill main fields in create product form" stepKey="basicComment" />`
It will print console message when test is running. And in case of test failure it will be easier to find a place where test is broken.